examples/generator-fullstack/.graphqlrc.yml

```diff
@@ -21,12 +17,31 @@ extensions:
       subCreate: false
       subUpdate: false
       subDelete: false
-      disableGen: false
-    folders:
-      model: ./model
-      resolvers: ./server/src/resolvers
-      schema: ./server/src/schema
-      client: ./client/src/graphql
+    ## Codegen plugins
+    plugins:
+        graphback-schema:
+          format: 'gql'
+          outputPath: ./server/src/schema
+          outputFileName: 'generated'
+        graphback-client:
+          format: 'ts'
+          outputPath: ./client/src/graphql
+        graphback-resolvers:
+          format: ts
+          outputPath: ./server/src/resolvers
+          types:
+            resolverRootLocation: ../../generated-types
+            resolverRootType: Resolvers
+    ## Knex DB Migration config
+    dbmigrate:
```

Comment:
I really struggling to find good name here. we can incorporate that for dbmigration and then have separate db configuration in the application.

@craicoverflow Looking for opinion on this as I would like to avoid renaming it.

Reply:
generally we had this feedback that db is confusing and we did not documented how to swap DB here. DBMigrate is more descriptive about intention but then it will be strange to use that inside example app

Reply:
Does the config parent strictly have to include the migrate/migrations keyword? Could we have something like:

I don't see how `DBMigrate` is more beneficial really.

- dbOptions
- dbSettings
...

Reply:
It is really defining where this data is used. We do not use it for any other reason than migration

Reply:
Coming back to this. Do you think we should be back to db as name?

Reply:
I prefer `dbMigrations` over `dbMigrate` and am fine with using that.

The db config data is not just used in migrations. Migrations might happen once while these settings are used ever time the server runs for making a connection to the database.